ebpf/c/tracer-ebpf.c

```diff
@@ -561,13 +440,13 @@ static int increment_udp_stats(struct sock* sk,
             return 0;
         }
 
-        increment_ipv4_stats(sk, status, pid_tgid, CONN_TYPE_UDP, send_bytes, recv_bytes, ts);
+        update_conn_stats(sk, status, pid_tgid, type, CONN_V4, send_bytes, recv_bytes, ts);
```

Comment:
I got tripped up on the name update_conn_stats vs increment_conn_stats is confusing because it's actually update_conn_stats that is doing the incrementing now.  

Maybe `increment_conn_stats` could have a better name like ` handle_message`.